apps/eth/rpc.py

```diff
@@ -23,7 +24,8 @@ def subscribe(self, project, version, url=Event.get_event_subscription_url(), in
             "project": project,
             "interval": interval,
             "eventNames": events or ["allEvents"],
-            "version": version
+            "version": version,
+            "blockheight": blockheight
```

Comment:
This is the incorrect parameter name for the Engine method.  Per the README in Engine, this should be `lastBlock`.